### ISSUE TITLE
App freezes on Android

### ISSUE THREAD

**Issue by oxilor** [zmxv/react-native-sound#462]
Hi!
On iOS words perfectly. But when I call "new Sound" on Android, app freezes and nothing logging to console. Has anyone had this problem?

```js
var whoosh = new Sound('whoosh.mp3', Sound.MAIN_BUNDLE, (error) => {
  if (error) {
    console.log('failed to load the sound', error);
  } else {
    console.log('loaded successfully ')
  }
});
```

**Comment by oxilor** [comment]
I check it in empty project and run on android device and I have this problem too. When I call new Sound my app freezes.

**Comment by oxilor** [comment]
Sorry. I don't know why, but this problem with my device (xiaomi redmi 4a). I reboot it and library became works well.

**Comment by adirzoari** [comment]
@oxilor  same issue in my xioami redmi 4a. any solution?

**Comment by oxilor** [comment]
@adirzoari I just reboot my device and everything was okay.

**Comment by adirzoari** [comment]
@oxilor  I reboot my device and still it freeze. weird.
if I delete the sound, everything works good.

**Comment by oxilor** [comment]
@adirzoari What about another apps? Are they playing without freeze?

**Comment by adirzoari** [comment]
yes. I have many apps with sound and everything works good

**Comment by oxilor** [comment]
@adirzoari try download app Lingain, add new word in "My vocabulary" and click to speaker. Is app freeze?

**Comment by adirzoari** [comment]
no. your app is not freeze. when i add new word I hear it and it works good.

**Comment by oxilor** [comment]
@adirzoari It's weird. Try to create empty project and install react-native-sound [manually](https://github.com/zmxv/react-native-sound/wiki/Installation#manual-installation-on-android). It should be works on Xiaomi Redmi 4a. I'm using react-native-sound [like this](https://github.com/zmxv/react-native-sound/issues/320#issuecomment-405019968).

**Comment by adirzoari** [comment]
thanks I will try. in your example, you used react-native-fs. for what? 
i did this way (in debug mode it sometimes works and I hear the sound, in the release it freeze)

```
import Sound from 'react-native-sound'

export const playSound = (sound) => {
    const music = new Sound(sound, Sound.MAIN_BUNDLE, (error) => {
		if (error) {
			console.log('failed to load the sound', error);
			return;
        }
        // music.setVolume(0.8)
		// loaded successfully, play			
		music.play((success) => {
			console.log(success)
			if (success) {
				console.log('successfully finished playing');
				music.release();
			} else {
				console.log('playback failed due to audio decoding errors');
			}
		});
	});

}
```
